website/docs/segments/languages/zig.mdx

```diff
@@ -19,7 +19,8 @@ import Config from "@site/src/components/Config.js";
     powerline_symbol: "\ue0b0",
     foreground: "#342311",
     background: "#ffad55",
-    template: " \ue6a9 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
+    template:
+      " \ue6a9 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
```

Comment:
@JanDeDobbeleer Is this newline added by mistake?

Reply:
ah no, it's the formatter that does this.